import.py

```diff
@@ -120,22 +120,26 @@ def main():
 
 
 
-def reprojectShapefile(f, inputDir, outputDir, srs):
+#NB: this now dissolves shapes and reprojects.  Should I rename?
```

Comment:
OK, done, along with a few comments and clarification of an output prompt. I think this should be ready to merge now.